Library/Formula/swiftgen.rb

```diff
@@ -0,0 +1,15 @@
+class Swiftgen < Formula
+  desc "Collection of Swift tools to generate Swift code"
+  homepage "https://github.com/AliSoftware/SwiftGen"
+  url "https://github.com/AliSoftware/SwiftGen/archive/0.4.1.tar.gz"
+  sha256 "a38b4bbd27eb183e73d2c1d5245f3134b8d688bc3095b538e52ed1ca0b85180b"
+  head "https://github.com/AliSoftware/SwiftGen.git", :branch => "master"
+
+  def install
+    rake "all[#{prefix},/bin]"
```

Comment:
Can you not do something like `rake "install", "INSTALL_PATH=#{bin}"` or similar? Is that syntax unsupported?